Feature: Make the graph UIs more usable (#12793)
OpenTTD/OpenTTD: src/lang/english.txt

STR_GRAPH_CARGO_PAYMENT_RATES_DAYS                              :{TINY_FONT}{BLACK}Days in transit
STR_GRAPH_CARGO_PAYMENT_RATES_SECONDS                           :{TINY_FONT}{BLACK}Seconds in transit
STR_GRAPH_CARGO_PAYMENT_RATES_TITLE                             :{TINY_FONT}{BLACK}Payment for delivering 10 units (or 10,000 litres) of cargo a distance of 20 squares

# Selector widget
STR_SELECTOR_WIDGET_ENABLE_ALL                                  :{BLACK}Show all

> STR_SELECTOR_WIDGET_DISABLE_ALL                                 :{BLACK}Hide all

Reviewer: I opened #12847 to standardize these button titles elsewhere, so (assuming other devs agree with me), this PR should follow that standard.

Also note that `All` comes before `None` in the other uses, so the buttons should swap positions.

```suggestion
STR_SELECTOR_WIDGET_ALL                                         :{BLACK}All
STR_SELECTOR_WIDGET_NONE                                        :{BLACK}None
```

Reviewer: Not quite resolved, I think you lost something in a rebase. 😉

Author: Sometimes I get confused in a rebase :smile:

> STR_SELECTOR_WIDGET_TOOLTIP_ENABLE_ALL                          :{BLACK}Display none of the items in the list

Reviewer: Since these are new strings, you might as well name them properly (all four strings).

Also, both say `None`. 😉 

```suggestion
STR_SELECTOR_WIDGET_TOOLTIP_ALL                                 :{BLACK}Display all of the items in the list
```

> STR_SELECTOR_WIDGET_TOOLTIP_ALL                                 :{BLACK}Display none of the items in the list

Reviewer: ```suggestion
STR_SELECTOR_WIDGET_TOOLTIP_ALL                                 :{BLACK}Display all of the items in the list
```

> STR_SELECTOR_WIDGET_TOOLTIP_ENABLE_ALL                          :{BLACK}Make all the items in the list visible

Reviewer: ```suggestion
STR_SELECTOR_WIDGET_TOOLTIP_ENABLE_ALL                          :{BLACK}Display all of the items in the list
```

> STR_SELECTOR_WIDGET_TOOLTIP_DISABLE_ALL                         :{BLACK}Make all the items in the list invisible

Reviewer: ```suggestion
STR_SELECTOR_WIDGET_TOOLTIP_DISABLE_ALL                         :{BLACK}Display none of the items in the list
```
